packages/muze-tooltip/src/strategy.js

```diff
@@ -32,7 +32,7 @@ const defaultTooltipFormatters = (type, formatter) => {
         [MeasureSubtype.CONTINUOUS]: value => formatter(value ? value.toFixed(2) : value),
         [DimensionSubtype.CATEGORICAL]: value => value
     };
-    return formatters[type];
+    return formatters[type] || formatters[DimensionSubtype.CATEGORICAL];
```

Comment:
isn't it a job of sanitizer / policy maker to handle this?

Reply:
Here, defaultTooltipFormatters method acts as a sanitizer which actually returns the appropriate formatter method based on a given field type.

Reply:
Okay then write
```javascript
const defaultTooltipFormatters = (type= formatters[DimensionSubtype.CATEGORICAL], formatter) => { ... }
```

instead of 
```javascript
return formatters[type] || formatters[DimensionSubtype.CATEGORICAL];
```

Reply:
Yes, that works